projects/plugins/jetpack/modules/likes/queuehandler.js

```diff
@@ -238,32 +230,19 @@ function JetpackLikesMessageListener( event ) {
 				const element = document.createElement( 'li' );
 				list.append( element );
 
-				if ( newLayout ) {
-					element.innerHTML = `
-					<a href="${ encodeURI( liker.profile_URL ) }" rel="nofollow" target="_parent" class="wpl-liker">
-						<img src="${ liker.avatar_URL }"
-							alt=""
-							style="width: 28px; height: 28px;" />
-						<span></span>
-					</a>
-				`;
-				} else {
-					element.innerHTML = `
-					<a href="${ encodeURI( liker.profile_URL ) }" rel="nofollow" target="_parent" class="wpl-liker">
-						<img src="${ liker.avatar_URL }"
-							alt=""
-							style="width: 30px; height: 30px; padding-right: 3px;" />
-					</a>
+				element.innerHTML = `
+				<a href="${ encodeURI( liker.profile_URL ) }" rel="nofollow" target="_parent" class="wpl-liker">
+					<img src="${ liker.avatar_URL }"
```

Comment:
I know we have already removed `encodeURI()` from the `avatar_URL` elsewhere - since this URL is already encoded. But thinking about this further, I will look at if we could make the `d=` part of the URL not encoded in the first place. As a result we would be able to keep the `encodeURI()` in place and send it URL that hasn't been encoded yet.

I will share my findings.

Anyways, I think we can still go ahead and merge this change as a part of this PR.

Reply:
> But thinking about this further, I will look at if we could make the d= part of the URL not encoded in the first place. As a result we would be able to keep the encodeURI() in place and send it URL that hasn't been encoded yet.
>
> I will share my findings.

Turns out we expect the Gravatar image URL is already encoded in other part of the codebase as well, e.g. emails. So I think we can leave the things as they are for now.

Reply:
Nice findings!
Code rebased @ivan-ottinger 👍

Reply:
Another rebase was necessary. I am reviewing it all now. 🙂